ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/handler/AbortedTxnCleaner.java

```diff
@@ -99,7 +100,7 @@ private void clean(CompactionInfo info, long minOpenWriteTxn, boolean metricsEna
       if (isNull(t)) {
         // The table was dropped before we got around to cleaning it.
         LOG.info("Unable to find table {}, assuming it was dropped.", info.getFullTableName());
-        txnHandler.markCleaned(info, true);
+        txnHandler.markCleaned((CompactionInfo) info, true);
```

Comment:
Is it possible to change `markCleaned` to take `AbortTxnInfo` instead of `CompactionInfo`?

Reply:
I'm not sure how Thrift will behave when old client interacts with new Server, or vice versa. Changing thrift signatures should be done in a separate PR as it can trigger further changes.

Reply:
Removed the AbortedTxnInfo class, it will be introduced together with the thrift changes